src/lib/intermediate-config-collector-builder.ts

```diff
@@ -0,0 +1,37 @@
+import { sortBy } from 'lodash';
```

Comment:
Could benefit from three-shaking as `import sortBy from 'lodash/sortBy';`.

Reply:
I believe webpack should be able to treeshake this anyway.

Reply:
In my experience it didn't.

Reply:
No worries, I changed it.
(A quick google shows that rollup can handle it better than webpack does.)